From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Sun, 26 Jun 2011 23:24:37 +0200
Subject: manpage dash slash

In manpages, need to guard dashes before options: not foo --help and
foo -h, but foo \-\-help and foo \-h.  Ask not what your computer can
do for you, but what you can do for your computer.
---
 src/man/man1/comdraw.1  | 10 +++++-----
 src/man/man1/drawtool.1 | 42 +++++++++++++++++++++---------------------
 src/man/man1/flipbook.1 |  6 +++---
 src/man/man3/Deck2_6.3  |  4 ++--
 src/man/man3/Rubband.3  |  4 ++--
 src/man/man3/UArray.3   |  2 +-
 src/man/man3/Viewer.3   |  2 +-
 src/man/man3/World.3    |  2 +-
 8 files changed, 36 insertions(+), 36 deletions(-)

diff --git a/src/man/man1/comdraw.1 b/src/man/man1/comdraw.1
index b359b43..4eed5d3 100644
--- a/src/man/man1/comdraw.1
+++ b/src/man/man1/comdraw.1
@@ -3,7 +3,7 @@
 comdraw \- drawtool with distributed command interpreter
 .SH SYNOPSIS
 .B comdraw 
-[-port n] [-import n] ['X-params'] [file]
+[\-port n] [\-import n] ['X-params'] [file]
 .SH DESCRIPTION
 comdraw is a drawing editor with integrated command interpreter the
 user can interact with via stdin and stdout (or via telnet if ACE is
@@ -161,7 +161,7 @@ barplot([var_str value_float] [...] :title title_str :xtitle xtitle_str
 .SH OTHER COMMANDS
 
  acknowledgebox(msgstr) -- popup an acknowledge dialog box
- confirmbox(msgstr) -- popup a confirmation dialog box, and return 1, 0, or -1 if cancelled
+ confirmbox(msgstr) -- popup a confirmation dialog box, and return 1, 0, or \-1 if cancelled
 
  run(filename) -- run commands from file
  quit() -- quit this interpreter
@@ -189,16 +189,16 @@ with ACE.
 processing.
 
 .B \-theight n,
-(or "-th n") selects the automatic raster tiling height.
+(or "\-th n") selects the automatic raster tiling height.
 
 .B \-twidth n,
-(or "-tw n") selects the automatic raster tiling width.
+(or "\-tw n") selects the automatic raster tiling width.
 
 .B \-tile,
  enables the automatic raster tiling.
 
 .PP
-Also see the -help message and the drawtool and idraw man pages for
+Also see the \-help message and the drawtool and idraw man pages for
 further options.
 
 .SH SEE ALSO  
diff --git a/src/man/man1/drawtool.1 b/src/man/man1/drawtool.1
index fbf67b0..a57daaf 100644
--- a/src/man/man1/drawtool.1
+++ b/src/man/man1/drawtool.1
@@ -3,7 +3,7 @@
 drawtool \- idraw with extensions
 .SH SYNOPSIS
 .B drawtool 
-.I -import n ['X-params'] [file]
+.I \-import n ['X-params'] [file]
 .SH DESCRIPTION
 drawtool is an extended version of idraw (originally from the
 InterViews 3.1).  Based on the ivtools OverlayUnidraw library, it adds
@@ -37,7 +37,7 @@ compress (.Z) and gzip (.gz) is recognized and handled for the script
 files themselves and the PBM, PGM, and PPM image formats.
 .PP
 There is also support for opening (via command line or dialog box) and
-importing via the net (to the -import port) any arbitrary PostScript
+importing via the net (to the \-import port) any arbitrary PostScript
 file using the pstoedit filter, and any GIF, TIFF, or JPEG image,
 using respectively the giftopnm, tifftopnm, or djpeg filters.
 .PP
@@ -63,70 +63,70 @@ multilines if drawtool has been built with the clippoly library
 
 .SH OPTIONS
 .TP
-"-import n" 
+"\-import n"
 specifies the port number to run the import service on.  The
 import service accepts connections over the net and reads drawtool
 or idraw documents and pbmplus image formats (PBM/PGM/PPM), plus JPEG, GIF,
 TIFF, and the non-raster portions of arbitrary PostScript if appropriate
 filters are available (djpeg, giftopnm, tifftopnm, pstoedit).
-"-color5" selects a colormap with 5 values per color or 125 entries (5
+"\-color5" selects a colormap with 5 values per color or 125 entries (5
 cubed).
 
 .PP
-"-nocolor6" overrides the selection of a colormap with 6 values per
+"\-nocolor6" overrides the selection of a colormap with 6 values per
 color or 216 entries (6 cubed).
 
 .PP
-"-gray5" selects a colormap with 5 bits of gray-level intensity (32
+"\-gray5" selects a colormap with 5 bits of gray-level intensity (32
 gray-levels) arranged to maximize rubberband visibility.
 
 .PP
-"-gray6" selects a colormap with 6 bits of gray-level intensity (64
+"\-gray6" selects a colormap with 6 bits of gray-level intensity (64
 gray-levels)  arranged to maximize rubberband visibility.
 
 .PP
-"-gray7" selects a colormap with 7 bits of gray-level intensity (128
+"\-gray7" selects a colormap with 7 bits of gray-level intensity (128
 gray-levels) arranged to maximize rubberband visibility.
 
 .PP
-"-dithermap" causes imported JPEG rasters to be mapped to the standard
-colormap (the default without -nocolor6) and dithered to avoid color
+"\-dithermap" causes imported JPEG rasters to be mapped to the standard
+colormap (the default without \-nocolor6) and dithered to avoid color
 banding.
 
 .PP
-"-opaque_off" (or "-opoff") disables the opaque (visible)
+"\-opaque_off" (or "\-opoff") disables the opaque (visible)
 interactive transformations of graphics.
 
 .PP
-"-pagecols" (or "-ncols") sets the horizontal page extent in pixels.
+"\-pagecols" (or "\-ncols") sets the horizontal page extent in pixels.
 
 .PP
-"-pagerows" (or "-nrows") sets the vertical page extent in pixels.
+"\-pagerows" (or "\-nrows") sets the vertical page extent in pixels.
 
 .PP
-"-panner_align tl|tc|tr|cl|c|cr|cl|bl|br|l|r|t|b|hc|vc" (or "-pal ...")
+"\-panner_align tl|tc|tr|cl|c|cr|cl|bl|br|l|r|t|b|hc|vc" (or "\-pal ...")
 selects the alignment of the panner display.
 
 .PP
-"-panner_off" (or "-poff") disables the panner display.
+"\-panner_off" (or "\-poff") disables the panner display.
 
 .PP
-"-slider_off" (or "-soff") disables the slider display.
+"\-slider_off" (or "\-soff") disables the slider display.
 
 .PP
-"-zoomer_off" (or "-zoff") disables the zoomer display.
+"\-zoomer_off" (or "\-zoff") disables the zoomer display.
 
 .PP
-"-ptrloc" enables display of the current pointer location.
+"\-ptrloc" enables display of the current pointer location.
 
 .PP
-"-scribble_pointer (or "-scrpt") enables continuous line drawing.
+"\-scribble_pointer (or "\-scrpt") enables continuous line drawing.
 
 .PP
-"-toolbarloc r|l" (or "-tbl ...") selects the toolbar location.
+"\-toolbarloc r|l" (or "\-tbl ...") selects the toolbar location.
 
 .PP
-Also see "-help" and the idraw man page for more options.
+Also see "\-help" and the idraw man page for more options.
 
 .SH SEE ALSO  
 	idraw(1)
diff --git a/src/man/man1/flipbook.1 b/src/man/man1/flipbook.1
index f390a6e..1bb0aa4 100644
--- a/src/man/man1/flipbook.1
+++ b/src/man/man1/flipbook.1
@@ -53,15 +53,15 @@ background.
 
 .SH OPTIONS
 
-"-bookgeom" brings up the flipbook in landscape mode, with a 906 by
+"\-bookgeom" brings up the flipbook in landscape mode, with a 906 by
 700 pixel page size.  Useful for viewing illustrated books prepared in
 this mode.
 
-"-slideshow sec" continuously plays the contents of a flipbook file
+"\-slideshow sec" continuously plays the contents of a flipbook file
 with "sec" delay between frames (currently only available when built
 with ACE).  
 
-Also see "-help" for more options, and the comdraw, drawtool, and
+Also see "\-help" for more options, and the comdraw, drawtool, and
 idraw man pages.
 
 .SH SEE ALSO
diff --git a/src/man/man3/Deck2_6.3 b/src/man/man3/Deck2_6.3
index 7a7e03e..540261a 100644
--- a/src/man/man3/Deck2_6.3
+++ b/src/man/man3/Deck2_6.3
@@ -52,7 +52,7 @@ the deck (the components inserted last), and a negative value specifies a
 component nearer to the ``top''.  FlipTo specifies a component by its
 absolute position in the deck.  A positive value of \fIposition\fP specifies
 a position from the top, starting at 1.  A negative value specifies a
-position from the bottom, starting at -1.  Top and Bottom are equivalent to
-FlipTo(1) and FlipTo(-1) respectively.
+position from the bottom, starting at \-1.  Top and Bottom are equivalent to
+FlipTo(1) and FlipTo(\-1) respectively.
 .SH SEE ALSO
 Interactor(3I), Scene(3I), Shape(3I)
diff --git a/src/man/man3/Rubband.3 b/src/man/man3/Rubband.3
index cb3efb4..43ba398 100644
--- a/src/man/man3/Rubband.3
+++ b/src/man/man3/Rubband.3
@@ -344,9 +344,9 @@ the tracking point.
 GrowingVertices is an abstract rubberband containing a list of
 vertices that grows or shrinks dynamically.  The \fIpt\fP parameter
 specifies the index of the vertex that will follow the tracking point;
-a value of -1 specifies a vertex after the last one in the list.  The
+a value of \-1 specifies a vertex after the last one in the list.  The
 \fIhandleSize\fP parameter specifies the size of the handle drawn at
-each vertex (as in RubberHandles above); a value of -1 specifies that
+each vertex (as in RubberHandles above); a value of \-1 specifies that
 no handles be drawn.
 
 The AddVertex operation adds a new vertex at the tracking point to the
diff --git a/src/man/man3/UArray.3 b/src/man/man3/UArray.3
index ae9da66..ea534c8 100644
--- a/src/man/man3/UArray.3
+++ b/src/man/man3/UArray.3
@@ -31,7 +31,7 @@ Insert and remove an object at a particular index, moving the
 succeeding objects up or down one position as required.
 .TP
 .B "int Index(void*)"
-Return the given object's index in the UArray, returning -1 if the
+Return the given object's index in the UArray, returning \-1 if the
 object does not appear in the UArray.
 .TP
 .B "int Count()"
diff --git a/src/man/man3/Viewer.3 b/src/man/man3/Viewer.3
index 13523bf..da2e3f2 100644
--- a/src/man/man3/Viewer.3
+++ b/src/man/man3/Viewer.3
@@ -105,7 +105,7 @@ editor's MagnifVar, if it defines one.
 Set and get the viewer's orientation parameter, which specifies
 whether the graphical view, page, and grid will appear in portrait or
 landscape mode.  No transformation will be applied in portrait mode,
-while landscape mode will rotate the page -90 degrees and translate it
+while landscape mode will rotate the page \-90 degrees and translate it
 vertically by an amount equal to the width of the page's bounding box.
 .TP
 .B "virtual Selection* GetSelection()"
diff --git a/src/man/man3/World.3 b/src/man/man3/World.3
index 297bd35..dc0d91b 100644
--- a/src/man/man3/World.3
+++ b/src/man/man3/World.3
@@ -20,7 +20,7 @@ It is currently provided for backward compatibility.
 .B ")"
 .br
 Construct the world object for a display.
-The world's instance name comes from the -name option if
+The world's instance name comes from the \-name option if
 it was given; otherwise the instance name comes from the environment
 variable RESOURCE_NAME if it is nonnil.  If the name has still not
 been defined, then argv[0] with all leading directories stripped is
